### DISCUSSION TITLE
[Kavita] Additional Fonts for ePub

### DISCUSSION THREAD

**Discussion by DieselTech** [Kareadita/Kavita#2550] [Ideas]
### Idea Description

Posted by Laria
· Nov 2021
· 17 Upvotes
https://feats.kavitareader.com/posts/10/additional-fonts-for-epub

Description

Either admin added or added by dev in folder. Suggestions: Crimson, Droid Serif, Georgia, OpenDyslexic, Palatino, Vollkorn, Roboto, Lucida, Helvetica, Noto (for various languages)

### Duration of Using Kavita

42

### Idea Category

Feature Enhancement

### Agreement

- [X] I agree that this is solely for submitting ideas, and I will search for existing ideas before posting.

**Comment by zhen-huan-hu** [comment]
I would like to be able to add my own fonts. My favorite font is Adobe Jenson Pro but it's a commercial font.

**Comment by majora2007** [comment]
This has been delivered as of nightly release (v0.8.7.17) and will be in the v0.8.8 stable release. 

#4037
